src/test/golden_buffers/BasicProcessingTestPause

```diff
@@ -1,512 +1,512 @@
-0.870045,0.870045
```

Comment:
can you explain why the tests are failing after your changes?

Reply:
The explanation is in the commit message:

rateRange is a percent expressed as a decimal (0 - 1 range), but the
test previously set rateRange to 4.0. Commit
29ef9d1 changed rateRange to enforce a
limited range between 0.01 and 0.90, which broke the incorrect test
setup.